internal/testrunner/runners/pipeline/runner.go

```diff
@@ -112,6 +112,10 @@ func (r *runner) run() ([]testrunner.TestResult, error) {
 		return nil, fmt.Errorf("listing test case definitions failed: %w", err)
 	}
 
+	if r.options.API == nil {
+		return nil, errors.New("missing elasticsearch client")
```

Comment:
Nit. Use consistent casing, in the other error message Kibana is upercased.
```suggestion
		return nil, errors.New("missing Elasticsearch client")
```